libs/template/labs/src/articles/resolvers/article-page-description-meta.resolver.ts

```diff
@@ -7,30 +7,27 @@ import {
 import { inject } from '@spryker-oryx/di';
 import { RouterService } from '@spryker-oryx/router';
 import { Observable, combineLatest, map, of, switchMap } from 'rxjs';
-
 import { ArticleContext } from '../article-context';
-import { CmsArticle } from '../article.model';
-import { ContentfulContentFields } from '../contentful';
-import { StoryblokContentFields } from '../storyblok';
+import { ArticleContent } from '../article.model';
 
 export class ArticlePageDescriptionMetaResolver implements PageMetaResolver {
   constructor(
     protected context = inject(ContextService),
-    protected router = inject(RouterService),
-    protected contentService = inject(ContentService)
+    protected content = inject(ContentService),
+    protected router = inject(RouterService)
   ) {}
 
   getScore(): Observable<unknown[]> {
     return combineLatest([
       this.context.get(document.body, ArticleContext.Id),
       this.context.get(document.body, ArticleContext.Type),
-      this.router
-        .currentRoute()
+      this.context
+        .get(document.body, ArticleContext.Type)
```

Comment:
```suggestion
        .get(null, ArticleContext.Type)
```
there is no document.body on the server side, so if you want to get fallback context, you should pass null